### PR TITLE
Add `county` as a new geoloc attribute

### PR DESCRIPTION
**Pull request by dsantosmerino**
It adds a new `geoloc` attribute called **county**. This geographical region is used in a lot of countries as you can see [here](https://en.wikipedia.org/wiki/County).

### CODE REVIEW

**Comment by dsantosmerino** [comment]
Adding this new attribute I realized that [Google's geocoder](https://github.com/geokit/geokit/blob/master/lib/geokit/geocoders/google.rb) is using the `administrive_area_level_2` to set the [district](https://github.com/geokit/geokit/blob/master/lib/geokit/geocoders/google.rb#L216-L217). Now that we have this new type may worth it to set the district value using the `sublocality` (working properly in France, Spain, US) or `administrative_area_level_3` (working properly in the UK). Are you interested as well in this changes?

**Comment by dsantosmerino** [comment]
Regarding the style offenses above. I can't find in the `.rubocop` where is set to prefer double-quoted.   Sure, I can change it, but I'd like to know where is configured 😄.

**Comment by rafaelpetry** [comment]
@dsantosmerino Just ran into the same issue with my PR. Looks like `.hound.yml` might be misconfigured, so it falls back to the default RuboCop config. According to [their docs](http://help.houndci.com/configuration/rubocop), it should look like:

```
rubocop:
  config_file: .rubocop.yml
```

the one on master looks like:

```
ruby:
  config_file: .rubocop.yml
```

File: lib/geokit/geo_loc.rb [hunk 1]
```diff
@@ -109,7 +110,7 @@ def formatted_address
     # gives you all the important fields as key-value pairs
     def hash
       res = {}
-      fields = [:success, :lat, :lng, :country_code, :city, :state, :zip,
+      fields = [:success, :lat, :lng, :country_code, :city, :county, :state, :zip,
```
[the remark this answers is not in the document]
Comment (dsantosmerino) [reply lib/geokit/geo_loc.rb]: Done!

File: lib/geokit/geo_loc.rb [hunk 2]
```diff
@@ -109,9 +110,9 @@ def formatted_address
     # gives you all the important fields as key-value pairs
     def hash
       res = {}
-      fields = [:success, :lat, :lng, :country_code, :city, :state, :zip,
-       :street_address, :district, :provider, :full_address, :is_us?,
-       :ll, :precision, :district_fips, :state_fips, :block_fips, :sub_premise]
+      fields = %i(success lat lng country_code city county state zip
```
[the remark this answers is not in the document]
Comment (dsantosmerino) [reply lib/geokit/geo_loc.rb]: Done!

**Pull request merged**